packages/web-app-files/src/components/AppBar/AppBar.vue

```diff
@@ -28,7 +28,7 @@
         </oc-breadcrumb>
         <shares-navigation v-if="hasSharesNavigation" />
         <div v-if="hasViewOptions || hasSidebarToggle" class="oc-flex">
-          <view-options v-if="hasViewOptions" />
+          <view-options v-if="hasViewOptions" :additional-view-modes="viewModeOptions" />
```

Comment:
I'd prefer `view-modes` as prop name and list all available view modes with it. Otherwise the next dev/reader needs to understand first what `additional` means.